### DISCUSSION TITLE
Making the Library, Page Info and cia. windowses respect set GTK theme on Linux?

### DISCUSSION THREAD

**Discussion by Impeta** [black7375/Firefox-UI-Fix#946] [Q&A] [answered]
Title says it all. Is it possible? Couldn't find an option related to it off UserChrome nor user.js. For visual reference, I want these;
![Screenshot_20240714_142409](https://github.com/user-attachments/assets/aa7402a6-a7cc-4027-a058-f365da274d7d)
![image](https://github.com/user-attachments/assets/59408f99-f063-4854-94cd-16eb1bd947d8)
![image](https://github.com/user-attachments/assets/07e42862-123c-4ba1-ad49-861659b82893)

Not these, which they came with when I got the Photon style.
![image](https://github.com/user-attachments/assets/d0e6c8b4-310f-41b8-965b-dc46820b6306)
![image](https://github.com/user-attachments/assets/89a40ab3-4b4c-4b04-a3a1-18772e631db9)
![image](https://github.com/user-attachments/assets/8e57f33c-2912-4a2b-93a7-80be7c51ffc1)

**Comment by black7375** [comment]
You can set `userContent.page.proton` to `false`

**Reply by Impeta** [reply]
Doesn't seem to work. Tried "Clean startup cache...", no dice either.
![Screenshot_20240717_121716](https://github.com/user-attachments/assets/7fcb0f70-05eb-4b2a-8e85-f181488b5f27)

**Reply by black7375** [reply, marked as answer]
Oh, I made a mistake.
Since it's about the browser window, you need to disable `userChrome.theme.proton_chrome`

**Reply by Impeta** [reply]
Oh that worked! Thanks!
Though another thing I noticed is the drop down menu off "Organize", "Views" and "Import and Backup" still look to default to plain Proton flair (and I assume any other menu off browser window), and I would like it preferably off as well. Can I disable that too or that's the most I bear with right now?
![image](https://github.com/user-attachments/assets/182bd0d1-f5a1-4153-952d-e52006274b63)

**Reply by black7375** [reply]
Of course you can.
Try to disable `userChrome.theme.non_native_menu`

![image](https://github.com/user-attachments/assets/f902c192-51d0-479e-a167-1685c9b1013a)
